frontend/src/components/Configuration/SensorsConfiguration/Barometer.jsx

```diff
@@ -50,30 +59,24 @@ export default function Barometer (sensor) {
                                 <FormControlLabel disabled control={<Switch checked={barometer.Enabled}  inputProps={{ 'aria-label': 'controlled' }} />}  label="Enabled" />
                             </FormGroup>
                         </Grid>
-                        {/* <Grid item xs={3}>
-                            <TextField id="Key" onChange={handleChange} label="Name" variant="standard" value={barometer.Key}/>
-                        </Grid> */}
-                        {/* <Grid item xs={3}>
-                            <TextField id="PressureFactorSigma" onChange={handleChange} label="PressureFactorSigma" variant="standard" value={barometer.PressureFactorSigma}/>
+                        <Grid item xs={3}>
+                            <TextField id="PressureFactorSigma" onChange={handleChange} label="Pressure Factor Sigma" type="number" variant="standard" value={barometer.PressureFactorSigma}/>
```

Comment:
bug here
1. cannot use up and down arrow to adjust the decimal value, 
2. cannot put in floating point number